src/test/java/com/pubnub/contract/ContractTestConfig.kt

```diff
@@ -26,6 +26,9 @@ interface ContractTestConfig : Config {
 
     @Config.Key("serverMock")
     fun serverMock(): Boolean
+
+    @Config.Key("personasLocation")
```

Comment:
Maybe `dataFilesLocation`? It's not only personas that we'll have there

Reply:
Yes, in face of working with Channels and Membership it make more sense. I will change it.